server/store/setting_store.go

```diff
@@ -8,9 +8,11 @@ import (
 const (
 	UpdateStatusSettingID               = "update_status"
 	GetConfirmationSettingID            = "get_confirmation"
-	ReceiveNotificationsDuringMeetingID = "receive_notification"
 	ReceiveRemindersSettingID           = "get_reminders"
 	DailySummarySettingID               = "summary_setting"
+	ReceiveNotificationsDuringMeetingID = "receive_notification"
```

Comment:
nit: would you mind putting this line were it was before? Just to clean the diff.